index.html

```diff
@@ -52,7 +52,271 @@ <h3>
         <li><a href='https://www.linkedin.com/in/fabriciolatl/'><img alt="linkedin" src='images/linkedin.svg' /></a></li> 
         <li><a href='https://twitter.com/Wafer_lt'><img alt="twitter" src='images/twitter.svg' /></a></li>
       </ul>
+
+      <section class="recent-works-container">
```

Comment:
Your work cards section also does not match the Figma template. Kindly fix it and then ask for a re-review.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/178111577-abb4ed2f-ab8a-4f62-acd7-a5f6d11080b5.png) | ![image](https://user-images.githubusercontent.com/33268927/178111612-62e2479a-dee8-4b91-b4e5-767604a1357c.png) |